hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -3894,6 +3894,18 @@
     </description>
   </property>
 
+  <property>
+    <name>ozone.om.snapshot.db.max.open.files</name>
+    <value>100</value>
+    <tag>OZONE, OM</tag>
+    <description>
+      Max number of open files for each snapshot db present in the snapshot cache.
```

Comment:
What if `snapshot db` is not added to cache?

Reply:
Currently we are loading all the snapshot db into the cache by design if we are doing any reads or writes.

Reply:
I think assumption that it will always be added to cache is wrong. Even if it is added to cache, there is no need to mention.